property_tests/arbitraries/candid/candid_arb.ts

```diff
@@ -1,16 +1,18 @@
 import fc from 'fast-check';
-import { deepEqual } from 'fast-equals';
 import { CandidType } from './candid_type_arb';
 
 // TODO we're thinking that Candid is not the best name for this. What is better?
-export type CandidMeta<T extends CandidType> = {
+export type CandidMeta<T extends CandidType, E = T> = {
     value: T;
```

Comment:
Can we think of a better name for this, somehow to distinguish these values? It's just not clear what the value is and why it's different than the expected value...do you think this is important?

For example should we say agentArgumentValue and agentResponseValue or something?

Reply:
Change these to argumentValue and expectedResponseValue